src/components/detail-process-instance/ExternalTask.vue

```diff
@@ -74,6 +75,22 @@ export default {
                     type: "error"
                 });
             })
+        },
+        unlockExtTask(extId) {
+            this.$api().post(`/external-task/${extId}/unlock`).then(response => {
+                this.$notify({
+                    group: "foo",
```

Comment:
foo? is it required?

Reply:
Not required, but this group used in all another notifications
https://www.npmjs.com/package/vue-notification#groups

Reply:
Without a group, the notification is not displayed

Reply:
it`s placeholder for notifications